utils/load_unitconv.m

```diff
@@ -42,12 +42,12 @@ function load_unitconv(ringmode, renamedIndexes)
     fprintf(f_units, '%d,%s,null,%d,%s,%s\n', renamedIndexes(db_indexes(i)), 'db0', 0, 'm^-1', 'A');
 end
 rfs = getfamilydata('RF');
-if length(rfs.ElementList) > 1
+if length(rfs.ElementList) == 1
```

Comment:
Do you need a separate case for length == 1?

Reply:
Yes, it doesn't have the AT field so we can't get its index from `rfs.AT.ATIndex(i)`